recipe/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "sse2neon" %}
+{% set version = "1.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DLTcollab/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 06f4693219deccb91b457135d836fc514a1c0a57e9fa66b143982901d2d19677
+  patches: # [not win]
+    - 0001-use-no-optimization-to-pass-all-tests.patch # [not win]
+  
+build:
+  number: 0
+  skip: True  # [s390x]
```

```suggestion
  skip: True  # [not aarch64 or arm64]
```
If the target is arm, it seems to me there is no point in building for other subdirs.

---

in the Makefile there is a support for intel x86, so I think the header can  be used as wrapper to run on both x86 & arm